### PR TITLE
Allow a file on the filesystem to be used as the source for the binary

### PR DESCRIPTION
**Pull request by groothuyse**
In my client's environment we are not able to retrieve the chromedriver over the network. We are however able to get the chromedriver binary on the filesystem. 

This change allows the installer to take a local file as the source instead of requiring a download. 
It would be awesome if this feature became available.

### CODE REVIEW

**Comment by fc** [comment]
Are you using Nighwatch or something similar?
You can define the path to chromedriver for Nightwatch using this:
https://github.com/nightwatchjs/nightwatch/wiki/Chrome-Setup

The 'webdriver.chrome.driver' setting is not specific to Nightwatch though as you can see here:
https://sites.google.com/a/chromium.org/chromedriver/getting-started

Would that work in your situation?

File: README.md [hunk 1]
```diff
@@ -47,6 +47,26 @@ Another option is to use PATH variable `CHROMEDRIVER_CDNURL`.
...
+Or add property into your [`.npmrc`](https://docs.npmjs.com/files/npmrc) file.
+
+```
+chromedriver_filepath=/path/to/chromedriver_mac64.zip
+```
+
+Another option is to use the PATH variable `CHROMEDRIVER_FILEPATH`
+
+```shell
+CHROMEDRIVER=/path/to/chromedriver_mac64.zip
```
Comment (giggio) [inline README.md]: This should probably be `CHROMEDRIVER_FILEPATH`, right?

Comment (groothuyse) [inline README.md]: Yeah you're totally right, I'll change that.

**Comment by giggio** [comment]
Published now as `2.27.2`.

**Pull request merged**